source/src/plugins/MeanWithinExpectedTest.cc

```diff
@@ -33,6 +33,8 @@
 #include <dqm4hep/QualityTest.h>
 #include <dqm4hep/StatusCodes.h>
 #include <dqm4hep/XmlHelper.h>
+#include <dqm4hep/AnalysisHelper.h>
```

Comment:
The file `MeanWithinExpectedTest.cc` and the test name should be renamed to PropertyWithinExpectedTest(.cc) as it is no longer the mean you can test